tree/ntuple/v7/src/RNTupleMerger.cxx

```diff
@@ -108,7 +108,10 @@ try {
                inFile->GetName());
          return -1;
       }
-      sources.push_back(RPageSourceFile::CreateFromAnchor(*anchor));
+      auto readOpts = RNTupleReadOptions();
+      // disable the cluster cache so we can catch the exception that happens on LoadEntry
+      readOpts.SetClusterCache(RNTupleReadOptions::EClusterCache::kOff);
```

Comment:
Should this stay as is? Background reading may be beneficial for merging.

Reply:
Sorry, I forgot to remove that line (it was for debugging)